docs/source/reference/websecurity.md

```diff
@@ -118,15 +118,20 @@ extend to securing the `tcp` sockets as well.
 
 ## Security audits
 
+<<<<<<< HEAD
+We recommend that you do periodic reviews of your deployment's security. It's
+good practice to keep [JupyterHub](https://readthedocs.org/projects/jupyterhub/), [configurable-http-proxy](https://github.com/jupyterhub/configurable-http-proxy), and [nodejs
+versions](https://github.com/nodejs/Release) up to date.
+=======
 We recommend that you do periodic reviews of your deployment's security. It is
 good practice to keep JupyterHub, configurable-http-proxy, and nodejs
 versions up to date.
 
+> > > > > > > main
```

Comment:
This looks like the remain of a merge conflict.

The following lines should be removed:

```
<<<<<<< HEAD
=======
> > > > > > > main
```

Can you then combine the two sentences that begin "We recommend that..." into a single sentence please?